src/scene/graphics/shared/utils/convertFillInputToFillStyle.ts

```diff
@@ -106,6 +140,62 @@ export function convertFillInputToFillStyle(
     style.color = color.toNumber();
     style.matrix = style.matrix ? style.matrix.clone() : null; // todo: lets optimise this!
 
-    // its a regular fill style!
     return style as ConvertedFillStyle;
 }
+
+export function convertFillInputToFillStyle<T extends FillInput>(
```

Comment:
I know this func name is old, but it is so verbose. What do you think about `toFillStyle` instead? Especially since you made this templated, it makes even less sense.

Reply:
Same for `convertStrokeInputToStrokeStyle` too. 

Also, can you add a doc block that explains _why_ this exists (private is final)? Just something about why are we need to do this conversion..

Reply:
yeah i did update the name to `toFillStyle` previously, then i reverted it to make the changes smaller. So more than happy to change to this again